main.go

```diff
@@ -117,12 +121,14 @@ func main() {
 			leaderElectionID = operatorName
 		}
 	}
-
+	// todo: How the schema is set to the manager?
 	options := ctrl.Options{
-		MetricsBindAddress:      "0.0.0.0:8383",
-		LeaderElection:          enableLeaderElection,
+		MetricsBindAddress: metricsAddr,
+		LeaderElection:     enableLeaderElection,
+		// todo: add:
+		Port:                    9443, // should not it be == kubebuilder?
 		LeaderElectionID:        leaderElectionID,
-		LeaderElectionNamespace: leaderElectionNamespace,
+		LeaderElectionNamespace: leaderElectionNamespace, // todo: why we need the ns?
```

Comment:
See; https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/main.go#L58

Reply:
The example is not working ; 

```
$ kubectl logs deployment.apps/nginx-operator-controller-manager -n nginx-operator-system -c manager
{"level":"info","ts":1592737343.092866,"logger":"setup","msg":"version information","go":"go1.13.10","GOOS":"linux","GOARCH":"amd64","helm-operator":"0.0.0+git"}
{"level":"info","ts":1592737343.0934982,"logger":"setup","msg":"watching all namespaces"}
{"level":"error","ts":1592737343.4005768,"logger":"setup","msg":"unable to start manager","error":"LeaderElectionID must be configured"}
```